tasks/tasks_prometheus_pgbouncer_exporter.yml

```diff
@@ -0,0 +1,58 @@
+---
+  - name: Set exporter facts
+    set_fact:
+      pgbouncer_exporter_sub_dir: "{{ exporter_config.exporter_dir | default('pgbouncer_exporter') }}"
+      pgbouncer_exporter_user: "{{ exporter_config.exporter_user | default(prometheus_user) }}"
+      pgbouncer_exporter_group: "{{ exporter_config.exporter_group | default(prometheus_group) }}"
+
+  - name: Prometheus pgbouncer exporter | ensure that python3 and dependencies are installed
+    become: yes
+    apt:
+      pkg:
+        - python3
+        - python3-pip
+
+  - name: Prometheus pgbouncer Exporter | install exporter with pip3
+    pip:
```

Comment:
I do not like explorer in global context. In most of situations virtualenv is preferable